database/migrations/2020_01_01_155000_create_results_table.php

```diff
@@ -22,7 +22,7 @@ public function up()
 
             // Don't need timestamps since can use created_at timestamp of the ranking class
 
-            $table->unique(['ranking_id', 'type', 'position'], 'result_unique');
+            //$table->unique(['ranking_id', 'type', 'position'], 'result_unique');
```

Comment:
I don't think sitelinks should prevent the use of unique results. They should have a different type value.

Reply:
I am not sure I understand. All the sitelinks of a website would have the same ranking id and position which they would get from the parent webpage. Should we have a "Sitelink" type enum for the "type" column? But even so, all sitelinks would have that same type. So the combination of the 3 would still be duplicated for the sitelinks.

Reply:
OK, I'm going to push this through and then let's take a look at how it runs in a Zoom meeting and I'll know more then what we should do. My thought is that I want to know what order the sitelinks are in. They should have a parent_id of the main result and be a different type with their own ranking.

Reply:
CC: @joshtorres @phuclh

Reply:
Oh! That makes sense! I think I get it now. However, sitelinks results don't have a "position" unlike local, organic, etc. We could use their position in the result set itself, or the "key" of the array element.
<img width="554" alt="Screen Shot 2021-04-12 at 11 18 23 PM" src="https://user-images.githubusercontent.com/64331952/114496373-cafc0300-9bed-11eb-94e9-0a0401efe133.png">

Reply:
Let's run some examples on their website and see if the key matches the order in which they are shown in the search results.